/usr/local/include/bind/arpa/inet.h:95:17: warning: '__inet_makeaddr' has C-linkage specified, but returns incomplete type 'struct in_addr' which could be incompatible with C [-Wreturn-type-c-linkage]
struct in_addr   inet_makeaddr __P((u_long , u_long));
                 ^
/usr/local/include/bind/arpa/inet.h:77:24: note: expanded from macro 'inet_makeaddr'
#define inet_makeaddr           __inet_makeaddr
                                ^

Index: src/net_util.h
--- src/net_util.h.orig
+++ src/net_util.h
@@ -26,9 +26,9 @@ enum IPFamily
 // breaks FreeBSD builds.
 // clang-format off
 #include <sys/types.h>
+#include <netinet/in.h>
 #include <arpa/inet.h>
 #include <cassert>
-#include <netinet/in.h>
 #include <netinet/in_systm.h>
 #include <netinet/ip.h>
 // clang-format on
